### ISSUE TITLE
[Camera Support]: Amiccom Z5

### ISSUE THREAD

**Issue by Dvalin21** [blakeblackshear/frigate#3750]
### Describe the problem you are having

FFmpeg is hating my camera to death. Can anyone suggest any config.yml change? Thank in advance.

### Version

DEBUG 0.10.1-83481AF

### Frigate config file

```yaml
mqtt:
  host: x.x.x.x
  port: 1883

detectors:
  cpu1:
    type: cpu
    num_threads: 3
  cpu2:
    type: cpu
    num_threads: 3

objects:
  track:
    - person
    - cat
    - dog
  filters:
    person:
      min_area: 5000
      max_area: 100000
      min_score: 0.5
      threshold: 0.7
      mask: 0,0,1000,0,1000,200,0,200

database:
  path: /media/frigate/frigate.db

ffmpeg:
  output_args:
    record: 
      -f segment 
      -segment_time 10 
      -segment_format mp4 
      -reset_timestamps 1 
      -strftime 1 
      -c:v libx264 -an
    rtmp: 
      -c:v libx264 
      -an
      -f flv

cameras:
  driveway:
    ffmpeg:
      inputs:
        - path: rtsp://pass:word@x.x.x.x:2600  <--- port 554 is blocked on the camera however this gives visual.
          roles:
           - detect
           - rtmp
           - record
    detect:
      width: 1280
      height: 720
      fps: 10
    record:
      enabled: True
      events:
        pre_capture: 5
        post_capture: 5
        retain:
          default: 10
    live:
      height: 1080
      quality: 2
```

### Relevant log output

```shell
[2022-08-31 18:36:14] frigate.app                    INFO    : Starting Frigate (0.10.1-83481af)
Starting migrations
[2022-08-31 18:36:14] peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
[2022-08-31 18:36:14] peewee_migrate                 INFO    : There is nothing to migrate
[2022-08-31 18:36:14] detector.cpu1                  INFO    : Starting detection process: 241
[2022-08-31 18:36:14] frigate.app                    INFO    : Output process started: 244
[2022-08-31 18:36:14] detector.cpu2                  INFO    : Starting detection process: 243
[2022-08-31 18:36:14] ws4py                          INFO    : Using epoll
[2022-08-31 18:36:14] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-08-31 18:36:14] frigate.app                    INFO    : Camera processor started for driveway: 247
[2022-08-31 18:36:14] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-08-31 18:36:14] frigate.app                    INFO    : Capture process started for driveway: 251
[2022-08-31 18:36:15] ws4py                          INFO    : Using epoll
[2022-08-31 18:37:22] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:34548]
[2022-08-31 18:37:25] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:40632]
[2022-08-31 18:37:33] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:40632]
[2022-08-31 18:38:25] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:34548]
[2022-08-31 18:56:55] watchdog.driveway              INFO    : No frames received from driveway in 20 seconds. Exiting ffmpeg...
[2022-08-31 18:56:55] watchdog.driveway              INFO    : Waiting for ffmpeg to exit gracefully...
[2022-08-31 18:56:56] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-08-31 18:56:56] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-08-31 18:57:06] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-08-31 18:57:06] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-08-31 18:57:06] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x55bd12a7adc0] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-08-31 18:57:06] ffmpeg.driveway.detect         ERROR   : More than 1000 frames duplicated
[2022-08-31 18:57:06] ffmpeg.driveway.detect         ERROR   : [flv @ 0x55bd12a9f100] Failed to update header with correct duration.
[2022-08-31 18:57:06] ffmpeg.driveway.detect         ERROR   : [flv @ 0x55bd12a9f100] Failed to update header with correct filesize.
[2022-08-31 18:57:56] watchdog.driveway              INFO    : No frames received from driveway in 20 seconds. Exiting ffmpeg...
[2022-08-31 18:57:56] watchdog.driveway              INFO    : Waiting for ffmpeg to exit gracefully...
[2022-08-31 18:57:57] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-08-31 18:57:57] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-08-31 18:58:07] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-08-31 18:58:07] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-08-31 18:58:07] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x555b673e1dc0] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-08-31 18:58:07] ffmpeg.driveway.detect         ERROR   : [flv @ 0x555b674079c0] Failed to update header with correct duration.
[2022-08-31 18:58:07] ffmpeg.driveway.detect         ERROR   : [flv @ 0x555b674079c0] Failed to update header with correct filesize.
[2022-08-31 18:59:07] watchdog.driveway              INFO    : No frames received from driveway in 20 seconds. Exiting ffmpeg...
[2022-08-31 18:59:07] watchdog.driveway              INFO    : Waiting for ffmpeg to exit gracefully...
[2022-08-31 18:59:09] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-08-31 18:59:09] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-08-31 18:59:19] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-08-31 18:59:19] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-08-31 18:59:19] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x559290971dc0] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-08-31 18:59:19] ffmpeg.driveway.detect         ERROR   : [h264 @ 0x559290b7d600] error while decoding MB 30 61, bytestream -8
[2022-08-31 18:59:19] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x559290971dc0] CSeq 6 expected, 0 received.
[2022-08-31 18:59:19] ffmpeg.driveway.detect         ERROR   : rtsp://pass:word@x.x.x.x:2600: corrupt decoded frame in stream 0
[2022-08-31 18:59:19] ffmpeg.driveway.detect         ERROR   : [flv @ 0x559290994140] Failed to update header with correct duration.
[2022-08-31 18:59:19] ffmpeg.driveway.detect         ERROR   : [flv @ 0x559290994140] Failed to update header with correct filesize.
```

### FFprobe output from your camera

```shell
ffprobe version 4.3.1 Copyright (c) 2007-2020 the FFmpeg developers
  built with gcc 9 (Ubuntu 9.3.0-17ubuntu1~20.04)
  configuration: --disable-debug --disable-doc --disable-ffplay --enable-shared --enable-avresample --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-gpl --enable-libfreetype --enable-libvidstab --enable-libmfx --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libxcb --enable-libx265 --enable-libxvid --enable-libx264 --enable-nonfree --enable-openssl --enable-libfdk_aac --enable-postproc --enable-small --enable-version3 --enable-libzmq --extra-libs=-ldl --prefix=/opt/ffmpeg --enable-libopenjpeg --enable-libkvazaar --enable-libaom --extra-libs=-lpthread --enable-vaapi --extra-cflags=-I/opt/ffmpeg/include --extra-ldflags=-L/opt/ffmpeg/lib
  libavutil      56. 51.100 / 56. 51.100
  libavcodec     58. 91.100 / 58. 91.100
  libavformat    58. 45.100 / 58. 45.100
  libavdevice    58. 10.100 / 58. 10.100
  libavfilter     7. 85.100 /  7. 85.100
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  7.100 /  5.  7.100
  libswresample   3.  7.100 /  3.  7.100
  libpostproc    55.  7.100 / 55.  7.100
x.x.x.x: No such file or directory
```

### Operating system

Other

### Install method

Docker Compose

### Coral version

CPU (no coral)

### Network connection

Wired

### Camera make and model

Amiccom Z5

**Comment by NickM-27** [comment]
Please try disabling rtmp

```yaml
rtmp:
  enabled: false
```

**Comment by Dvalin21** [comment]
> 

No change from what I can see

ConnectionResetError: [Errno 104] Connection reset by peer
[2022-09-01 11:29:31] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:49712]
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[2022-09-01 11:29:43] frigate.app                    INFO    : Starting Frigate (0.10.1-83481af)
Starting migrations
[2022-09-01 11:29:43] peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
[2022-09-01 11:29:43] peewee_migrate                 INFO    : There is nothing to migrate
[2022-09-01 11:29:43] detector.cpu1                  INFO    : Starting detection process: 241
[2022-09-01 11:29:43] frigate.app                    INFO    : Output process started: 244
[2022-09-01 11:29:43] detector.cpu2                  INFO    : Starting detection process: 243
[2022-09-01 11:29:43] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-09-01 11:29:43] frigate.app                    INFO    : Camera processor started for driveway: 247
[2022-09-01 11:29:43] ws4py                          INFO    : Using epoll
[2022-09-01 11:29:43] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-09-01 11:29:43] frigate.app                    INFO    : Capture process started for driveway: 250
[2022-09-01 11:29:43] ws4py                          INFO    : Using epoll
[2022-09-01 11:29:49] frigate.record                 WARNING : Discarding a corrupt recording segment: {'cache_path': '/tmp/cache/driveway-20220901112657.mp4', 'start_time': datetime.datetime(2022, 9, 1, 11, 26, 57)}
[2022-09-01 11:29:55] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:49770]
[2022-09-01 11:30:24] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:49770]
[2022-09-01 11:30:25] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:43722]
[2022-09-01 11:30:25] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:49798]
[2022-09-01 11:30:39] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:49798]
[2022-09-01 11:30:44] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:49872]
[2022-09-01 11:31:53] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:43722]
[2022-09-01 11:56:55] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:49872]
[2022-09-01 12:25:22] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-01 12:25:22] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-01 12:25:23] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-01 12:25:23] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-01 12:25:23] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x56545624bb80] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-01 12:25:23] ffmpeg.driveway.detect         ERROR   : More than 1000 frames duplicated
[2022-09-01 12:26:33] watchdog.driveway              INFO    : No frames received from driveway in 20 seconds. Exiting ffmpeg...
[2022-09-01 12:26:33] watchdog.driveway              INFO    : Waiting for ffmpeg to exit gracefully...
[2022-09-01 12:26:39] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-01 12:26:39] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-01 12:26:49] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-01 12:26:49] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-01 12:26:49] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x563b08d40b80] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-01 12:26:55] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-01 12:26:55] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-01 12:26:59] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-01 12:26:59] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-01 12:26:59] ffmpeg.driveway.detect         ERROR   : rtsp://pass:word@x.x.x.x:2600: Invalid data found when processing input
[2022-09-01 13:27:20] watchdog.driveway              INFO    : No frames received from driveway in 20 seconds. Exiting ffmpeg...
[2022-09-01 13:27:20] watchdog.driveway              INFO    : Waiting for ffmpeg to exit gracefully...
[2022-09-01 13:27:20] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-01 13:27:20] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-01 13:27:30] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-01 13:27:30] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-01 13:27:30] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x5555b3538b80] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-01 13:27:30] ffmpeg.driveway.detect         ERROR   : More than 1000 frames duplicated
[2022-09-01 13:28:30] watchdog.driveway              INFO    : No frames received from driveway in 20 seconds. Exiting ffmpeg...
[2022-09-01 13:28:30] watchdog.driveway              INFO    : Waiting for ffmpeg to exit gracefully...
[2022-09-01 13:28:36] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-01 13:28:36] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-01 13:28:46] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-01 13:28:46] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-01 13:28:46] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x558fa8aafb80] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-01 13:43:36] watchdog.driveway              INFO    : No frames received from driveway in 20 seconds. Exiting ffmpeg...
[2022-09-01 13:43:36] watchdog.driveway              INFO    : Waiting for ffmpeg to exit gracefully...
[2022-09-01 13:43:37] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-01 13:43:37] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-01 13:43:47] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-01 13:43:47] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-01 13:43:47] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x560ad6472b80] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-01 14:18:45] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:51170]
[2022-09-01 14:18:57] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:51170]
[2022-09-01 14:19:02] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:45170]
[2022-09-01 14:20:54] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:45170]
[2022-09-01 14:29:25] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-01 14:29:25] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-01 14:29:27] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-01 14:29:27] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-01 14:29:27] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x55ee81ecab80] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-01 14:29:27] ffmpeg.driveway.detect         ERROR   : More than 1000 frames duplicated
[2022-09-01 14:30:27] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-01 14:30:27] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-01 14:30:27] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-01 14:30:27] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-01 14:30:27] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x561d97134b80] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-01 14:30:30] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-01 14:30:30] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-01 14:30:37] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-01 14:30:37] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-01 14:30:37] ffmpeg.driveway.detect         ERROR   : [tcp @ 0x55565f529480] Connection to tcp://x.x.x.x:2600?timeout=5000000 failed: No route to host
[2022-09-01 14:30:37] ffmpeg.driveway.detect         ERROR   : rtsp://pass:word@x.x.x.x:2600: No route to host
[2022-09-01 14:38:44] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:51458]
[2022-09-01 14:38:45] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:51458]
[2022-09-01 14:38:55] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:51466]
[2022-09-01 14:38:56] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:51466]

**Comment by NickM-27** [comment]
The logs are entirely different and no longer giving `flv` errors. 

At this point looks like ffmpeg doesn't see your camera at all as it says there is no route to it. 

Are you that the cameras IP is correct and there's no firewall blocking frigate from communicating with it?

**Comment by Dvalin21** [comment]
> The logs are entirely different and no longer giving `flv` errors.
> 
> At this point looks like ffmpeg doesn't see your camera at all as it says there is no route to it.
> 
> Are you that the cameras IP is correct and there's no firewall blocking frigate from communicating with it?

Thanks for that, and your right the flv is not there. Im pretty sure that the ip is right as im able to view the feed from the camera. I think what could be messing things up is that frigate is expecting rtsp to be port 554. The manufacturer has blocked that port and opened port 2600. So this could be the issue. Also, this camera is a ONVIF. It functions on port 80. I did a scan on it and it came back saying " no authentication found on "/" I believe this is how the app is able to access the camera and set the setting. But many software has an issue with it as i think they are looking for authentication. Anyway, I just wanted to throw that in there just in case it helps. Is there anything else you want me try?

**Comment by NickM-27** [comment]
As long as you set the actual port correctly, ffmpeg will work with it so that isn't the issue. 

Based on the log `ffmpeg.driveway.detect ERROR : [rtsp @ 0x560ad6472b80] getaddrinfo(IP4 127.0.0.1): Name or service not known`

you seem to be setting the IP address for the camera as `127.0.01` which is localhost. Of course frigate will not be able to access that device with an internal IP address since that camera is not inside of frigate's internal network. You will need to use the actual IP address that is resolvable from within frigate.

**Comment by Dvalin21** [comment]
How do I do that?  If you look at the config.yml  I have rtsp://username:password@ip address:2600  So is there another place Where I can change it?

**Comment by NickM-27** [comment]
You have masked out your config, but those logs tell me that you have set 127.0.0.1 as the camera's IP address, which is incorrect. You would need to make that the right one (ie not an internal network one). I might be mistaken on that being what your, but the logs seem to indicate that and your config is masked out in that part

**Comment by Dvalin21** [comment]
Ok so how do I change the right one?  Im not even sure where the 127.0.0.1 is coming from at this point. Can't remember if I mentioned, but im running Frigate in a docker container on Synology Nas

**Comment by NickM-27** [comment]
You need to answer my question, what IP do you have for the camera in your frigate config? Is it 127.0.0.1? The log I put above indicates to me that it is, this is what I need to know to be able to move forward with helping you.

**Comment by Dvalin21** [comment]
> You need to answer my question, what IP do you have for the camera in your frigate config? Is it 127.0.0.1? The log I put above indicates to me that it is, this is what I need to know to be able to move forward with helping you.

10.0.0.2 (Ip which is from the router)

**Comment by NickM-27** [comment]
Okay I see, so the log wasn't saying the devices IP. 

Try adding `/stream0` to the end, based on some googling that is required

**Comment by NickM-27** [comment]
> Waiting for portainer to throw something at me.

What do you mean? Like error logs? Should be able to see if it worked with the frigate web UI

**Comment by Dvalin21** [comment]
[2022-09-02 14:22:05] frigate.app                    INFO    : Starting Frigate (0.10.1-83481af)
Starting migrations
[2022-09-02 14:22:05] peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
[2022-09-02 14:22:05] peewee_migrate                 INFO    : There is nothing to migrate
[2022-09-02 14:22:05] detector.cpu1                  INFO    : Starting detection process: 241
[2022-09-02 14:22:05] frigate.app                    INFO    : Output process started: 244
[2022-09-02 14:22:05] detector.cpu2                  INFO    : Starting detection process: 243
[2022-09-02 14:22:05] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-09-02 14:22:05] ws4py                          INFO    : Using epoll
[2022-09-02 14:22:05] frigate.app                    INFO    : Camera processor started for driveway: 249
[2022-09-02 14:22:05] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-09-02 14:22:05] frigate.app                    INFO    : Capture process started for driveway: 251
[2022-09-02 14:22:05] ws4py                          INFO    : Using epoll
[2022-09-02 14:22:11] frigate.record                 WARNING : Discarding a corrupt recording segment: {'cache_path': '/tmp/cache/driveway-20220902141921.mp4', 'start_time': datetime.datetime(2022, 9, 2, 14, 19, 21)}
[2022-09-02 14:23:27] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:56078]
[2022-09-02 14:23:29] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:33924]
[2022-09-02 14:23:39] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:33924]
[2022-09-02 14:23:39] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:33928]
[2022-09-02 14:23:43] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:33928]
[2022-09-02 14:34:15] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:56078]
[2022-09-02 14:50:40] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-02 14:50:40] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-02 14:50:45] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-02 14:50:45] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-02 14:50:45] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x5586ca0e5b80] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-02 14:50:45] ffmpeg.driveway.detect         ERROR   : [h264 @ 0x5586ca1d4380] error while decoding MB 113 3, bytestream -5
[2022-09-02 14:50:45] ffmpeg.driveway.detect         ERROR   : rtsp://10.0.0.2:2600/stream0: corrupt decoded frame in stream 0
[2022-09-02 14:50:45] ffmpeg.driveway.detect         ERROR   : More than 1000 frames duplicated
[2022-09-02 14:50:45] ffmpeg.driveway.detect         ERROR   : [h264 @ 0x5586ca1dde40] error while decoding MB 71 11, bytestream -9
[2022-09-02 14:50:45] ffmpeg.driveway.detect         ERROR   : rtsp://admin:Lolo1002@10.0.0.2:2600/stream0: corrupt decoded frame in stream 0
[2022-09-02 14:52:25] watchdog.driveway              INFO    : No frames received from driveway in 20 seconds. Exiting ffmpeg...
[2022-09-02 14:52:25] watchdog.driveway              INFO    : Waiting for ffmpeg to exit gracefully...
[2022-09-02 14:52:26] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-02 14:52:26] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-02 14:52:36] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-02 14:52:36] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-02 14:52:36] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x55a26c26ab80] getaddrinfo(IP4 127.0.0.1): Name or service not known

**Comment by NickM-27** [comment]
I just saw you asked about audio, please see: https://docs.frigate.video/faqs#audio-in-recordings

Does your password have any special characters in it? This can cause issues with ffmpeg

**Comment by Dvalin21** [comment]
> I just saw you asked about audio, please see: https://docs.frigate.video/faqs#audio-in-recordings
> 
> Does your password have any special characters in it? This can cause issues with ffmpeg

Thanks for the direction, I just went to that for the audio before I saw this. No, there are no special characters in the password.

**Comment by NickM-27** [comment]
Can you please try the url exactly as you have it in the config (with /stream0) and try it in VLC, report if it works and also a screenshot of the codec information

**Comment by Dvalin21** [comment]
> Can you please try the url exactly as you have it in the config (with /stream0) and try it in VLC, report if it works and also a screenshot of the codec information

I did try with the stream0 and the logs I shared was from the result. I didn't try it in vlc, ill do it know. However I will say that I have tried in vlc without the /stream0 and it worked.

**Comment by Dvalin21** [comment]
![codec](https://user-images.githubusercontent.com/7469013/188235262-54dfeccc-bb85-4df2-bd88-4cce454ce53d.jpg)
So it worked this time...Also attached is the codec info

**Comment by NickM-27** [comment]
Okay please remove 

```yaml
ffmpeg:
  output_args:
    record: 
      -f segment 
      -segment_time 10 
      -segment_format mp4 
      -reset_timestamps 1 
      -strftime 1 
      -c:v libx264 -an
    rtmp: 
      -c:v libx264 
      -an
      -f flv
```

from your config and try again, your stream is standard h264 and should work with default settings and I believe those settings are throwing it off

**Comment by Dvalin21** [comment]
> Okay please remove
> 
> ```yaml
> ffmpeg:
>   output_args:
>     record: 
>       -f segment 
>       -segment_time 10 
>       -segment_format mp4 
>       -reset_timestamps 1 
>       -strftime 1 
>       -c:v libx264 -an
>     rtmp: 
>       -c:v libx264 
>       -an
>       -f flv
> ```
> 
> from your config and try again, your stream is standard h264 and should work with default settings and I believe those settings are throwing it off

Ok thanks. I'm assuming that I will still need to add the -c:a acc for audio correct?

**Comment by NickM-27** [comment]
> Ok thanks. I'm assuming that I will still need to add the -c:a acc for audio correct?

Let's start with just getting the camera stream to work and worry about adding audio once that is done successfully

**Comment by Dvalin21** [comment]
> > Ok thanks. I'm assuming that I will still need to add the -c:a acc for audio correct?
> 
> Let's start with just getting the camera stream to work and worry about adding audio once that is done successfully


[2022-09-02 16:34:55] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:34:57] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-02 16:34:57] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-02 16:34:57] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x55be7d7c9900] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-02 16:34:57] ffmpeg.driveway.detect         ERROR   : File '/tmp/cache/driveway-%Y%m%d%H%M%S.mp4' already exists. Overwrite? [y/N] Not overwriting - exiting
[2022-09-02 16:35:00] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:35:00] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:35:05] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:35:05] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:35:05] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-02 16:35:05] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-02 16:35:07] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-02 16:35:07] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-02 16:35:07] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x55ffc72fa900] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-02 16:35:07] ffmpeg.driveway.detect         ERROR   : File '/tmp/cache/driveway-%Y%m%d%H%M%S.mp4' already exists. Overwrite? [y/N] Not overwriting - exiting
[2022-09-02 16:35:10] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-02 16:35:10] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-02 16:35:10] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:35:10] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:35:15] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:35:15] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:35:17] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-02 16:35:17] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-02 16:35:17] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x55742f6d6900] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-02 16:35:17] ffmpeg.driveway.detect         ERROR   : File '/tmp/cache/driveway-%Y%m%d%H%M%S.mp4' already exists. Overwrite? [y/N] Not overwriting - exiting
[2022-09-02 16:35:20] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-02 16:35:20] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-02 16:35:20] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:35:20] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:35:25] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:35:25] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:35:27] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-02 16:35:27] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-02 16:35:27] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x5589f2ef3900] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-02 16:35:27] ffmpeg.driveway.detect         ERROR   : File '/tmp/cache/driveway-%Y%m%d%H%M%S.mp4' already exists. Overwrite? [y/N] Not overwriting - exiting
[2022-09-02 16:35:30] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-02 16:35:30] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-02 16:35:30] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:35:30] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:35:35] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:35:35] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:35:37] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-02 16:35:37] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-02 16:35:37] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x557453ae5900] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-02 16:35:37] ffmpeg.driveway.detect         ERROR   : File '/tmp/cache/driveway-%Y%m%d%H%M%S.mp4' already exists. Overwrite? [y/N] Not overwriting - exiting
[2022-09-02 16:35:40] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:35:40] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:35:43] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-02 16:35:43] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-02 16:35:45] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:35:45] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:35:47] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-02 16:35:47] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-02 16:35:47] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x55a47a9df900] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-02 16:35:47] ffmpeg.driveway.detect         ERROR   : File '/tmp/cache/driveway-%Y%m%d%H%M%S.mp4' already exists. Overwrite? [y/N] Not overwriting - exiting
[2022-09-02 16:35:50] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-02 16:35:50] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-02 16:35:50] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:35:50] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:35:55] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:35:55] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:35:57] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-02 16:35:57] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-02 16:35:57] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x564466d76900] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-02 16:35:57] ffmpeg.driveway.detect         ERROR   : File '/tmp/cache/driveway-%Y%m%d%H%M%S.mp4' already exists. Overwrite? [y/N] Not overwriting - exiting
[2022-09-02 16:36:00] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-02 16:36:00] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-02 16:36:00] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:36:00] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:36:02] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:37002]
[2022-09-02 16:36:05] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:36:05] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:36:07] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-02 16:36:07] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-02 16:36:07] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x5609d0fcf900] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-02 16:36:07] ffmpeg.driveway.detect         ERROR   : File '/tmp/cache/driveway-%Y%m%d%H%M%S.mp4' already exists. Overwrite? [y/N] Not overwriting - exiting
[2022-09-02 16:36:10] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-02 16:36:10] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-02 16:36:10] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:36:10] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:36:15] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:36:15] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:36:17] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-02 16:36:17] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-02 16:36:17] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x5591b55c2900] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-02 16:36:17] ffmpeg.driveway.detect         ERROR   : File '/tmp/cache/driveway-%Y%m%d%H%M%S.mp4' already exists. Overwrite? [y/N] Not overwriting - exiting
[2022-09-02 16:36:20] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:36:20] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:36:25] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:36:25] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'
[2022-09-02 16:36:26] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-02 16:36:26] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-02 16:36:27] watchdog.driveway              ERROR   : Ffmpeg process crashed unexpectedly for driveway.
[2022-09-02 16:36:27] watchdog.driveway              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
[2022-09-02 16:36:27] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x5586fb522900] getaddrinfo(IP4 127.0.0.1): Name or service not known
[2022-09-02 16:36:27] ffmpeg.driveway.detect         ERROR   : File '/tmp/cache/driveway-%Y%m%d%H%M%S.mp4' already exists. Overwrite? [y/N] Not overwriting - exiting
[2022-09-02 16:36:30] frigate.video                  ERROR   : driveway: Unable to read frames from ffmpeg process.
[2022-09-02 16:36:30] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-02 16:36:30] frigate.record                 ERROR   : Error occurred when attempting to maintain recording cache
[2022-09-02 16:36:30] frigate.record                 ERROR   : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'

**Comment by NickM-27** [comment]
Please paste your new config

**Comment by Dvalin21** [comment]
> Please paste your new config

I will, I had to leave the house. It may be awhile. I'll post it as soon as I get home.

**Comment by Dvalin21** [comment]
I can tell ya the camera is not showing any feed anymore. It could be when I removed the -c:v libx264

**Comment by NickM-27** [comment]
That command tells it that it needs to transcode to h264 and is definitely not needed for your camera as it is standard h264 already. 

But hold on I missed that detail, I didn't realize the camera was originally working to some extent. Can you elaborate on how it was behaving before?

**Comment by Dvalin21** [comment]
> That command tells it that it needs to transcode to h264 and is definitely not needed for your camera as it is standard h264 already.
> 
> But hold on I missed that detail, I didn't realize the camera was originally working to some extent. Can you elaborate on how it was behaving before?

Ok, sorry its late. I wanted to make sure that I'm fine tuning everything. Trying to make sure ffmpeg is actually working. By all rights, it should be giving me the errors that Its giving me with the settings I got.

**Comment by NickM-27** [comment]
Sorry maybe I'm not understanding, what's the current status of things?

**Comment by Dvalin21** [comment]
> Sorry maybe I'm not understanding, what's the current status of things?

Sorry, Im trying to eliminate the ffmpeg errors that I'm getting. I would like to maximize the functionality with this camera.

**Comment by NickM-27** [comment]
> > Sorry maybe I'm not understanding, what's the current status of things?
> 
> Sorry, Im trying to eliminate the ffmpeg errors that I'm getting. I would like to maximize the functionality with this camera.

Yes, what is the current behavior and what was the behavior previously (you said it was somewhat working)

**Comment by Dvalin21** [comment]
So I'm able to detect, I'm able to see the feed. I'm able to record, but for some reason it only records for a short period instead of the full 300 secs. I've also set it for 15 start before and after. Also there are the times the feed will just go white and stay white until I refresh.

**Comment by NickM-27** [comment]
> but for some reason it only records for a short period instead of the full 300 secs

I think you're misunderstanding something, Frigate doesn't record for a default amount of time for events. It will only save recordings as long as the object that was detected for the event is visible and detected.

> Also there are the times the feed will just go white and stay white until I refresh.

What do you mean by refresh? Like refresh the browser or restart frigate?

**Comment by Dvalin21** [comment]
> > but for some reason it only records for a short period instead of the full 300 secs
> 
> I think you're misunderstanding something, Frigate doesn't record for a default amount of time for events. It will only save recordings as long as the object that was detected for the event is visible and detected.
> 
> > Also there are the times the feed will just go white and stay white until I refresh.
> 
> What do you mean by refresh? Like refresh the browser or restart frigate?

Refresh Browser

**Comment by Dvalin21** [comment]
The another issue is that when clip is recorded, it looks like pixels late or seems like its ghosting.

**Comment by Dvalin21** [comment]
[2022-09-04 20:42:32] frigate.video                  ERROR   : driveway: ffmpeg process is not running. exiting capture thread...
[2022-09-04 18:04:13] ffmpeg.driveway.detect         ERROR   : [rtsp @ 0x5592c6bddb80] Could not find codec parameters for stream 0 (Video: h264, none): unspecified size

**Comment by Dvalin21** [comment]
Final config.yml  This seems to be the most table. I believe the other issue (blurred ghosting pixel) are the same. Im thinking that is the result of the camera itself. I did get audio working with code below.  Thanks for the help @NickM-27 

mqtt:
  host: x.x.x.x
  port: 1883

detectors:
  cpu1:
    type: cpu
    num_threads: 3
  cpu2:
    type: cpu
    num_threads: 3

objects:
  track:
    - person
    - cat
    - dog
  filters:
    person:
      min_area: 5000
      max_area: 100000
      min_score: 0.5
      threshold: 0.7
      mask: 
        - 902,1080,791,0,0,0,0,1080
        - 1920,0,1920,147,870,274,819,0
        - 1537,222,1582,438,1920,500,1920,175
    cat:
      min_area: 5000
      max_area: 100000
      min_score: 0.5
      threshold: 0.7
      mask: 
        - 902,1080,791,0,0,0,0,1080
        - 1920,0,1920,147,870,274,819,0
        - 1537,222,1582,438,1920,500,1920,175
    dog:
      min_area: 5000
      max_area: 100000
      min_score: 0.5
      threshold: 0.7
      mask: 
        - 902,1080,791,0,0,0,0,1080
        - 1920,0,1920,147,870,274,819,0
        - 1537,222,1582,438,1920,500,1920,175
motion:
  threshold: 25
  contour_area: 30
  delta_alpha: 0.2
  frame_alpha: 0.2
  frame_height: 50
  mask:
    - 902,1080,791,0,0,0,0,1080
    - 1920,0,1920,147,870,274,819,0
    - 1537,222,1582,438,1920,500,1920,175
  improve_contrast: False

database:
  path: /media/frigate/frigate.db

logger:
  default: info
  logs:
    frigate.event: debug

ffmpeg:   
  output_args:
    record: 
      -f segment 
      -segment_time 10 
      -segment_format mp4 
      -reset_timestamps 1 
      -strftime 1 
      -acodec aac
    rtmp:
      -acodec aac

cameras:
  driveway:
    ffmpeg:
      inputs:
        - path: rtsp://pass:word@10.0.0.2:2600/ch0/stream0/live
          roles:
           - detect
           - rtmp
           - record
    detect:
      width: 1920
      height: 1080
      fps: 9
    motion:
      mask:
       - 902,1080,791,0,0,0,0,1080
       - 1920,0,1920,147,870,274,819,0
       - 1537,222,1582,438,1920,500,1920,175
    record:
      enabled: True
      events:
        pre_capture: 15
        post_capture: 15
        retain:
          default: 7
    rtmp:
      enabled: False
live:
  height: 720
  quality: 1
